### PR TITLE
Feature/materialize UI

### PR DESCRIPTION
**Pull request by tumophiy**
Completed work:
- add gem materialize-css;
- add necessary setup for this library to work.

As for now all views are being 'materialized'. I am not sure what is our current status about this library, cause I missed meeting at Monday. Are we using it?

[Trello](https://trello.com/c/Je04v59y) card.

### CODE REVIEW

File: app/javascript/packs/application.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ require("@rails/ujs").start()
 require("turbolinks").start()
 require("@rails/activestorage").start()
 require("channels")
+//= require turbolinks
```
Comment (igorkasyanchuk) [inline app/javascript/packs/application.js:10]: do you need these comments?
do you know that lib turbolinks is obsolete now, it was changed to "turbo"

Comment (mmrshk) [reply app/javascript/packs/application.js:10]: @igorkasyanchuk https://github.com/turbolinks/turbolinks
Screenshot 2022-07-03 at 07 25 04
Can you please provide some links for it? Cause, official docs shows that we still need to write - turbolinks

Comment (igorkasyanchuk) [reply app/javascript/packs/application.js:10]: image

and https://guides.rubyonrails.org/working_with_javascript_in_rails.html#turbo-drive

Comment (igorkasyanchuk) [reply app/javascript/packs/application.js:10]: @mmrshk ⬆️

**Comment by manastyretskyi** [comment]
Soon it would be two weeks since the PR was opened and no changes were made since, what's with this PR, why is it stale?
CC @mmrshk

**Review by bagiratania98 — APPROVED** [review]
LGTM

**Review by MishaKlymenko — APPROVED** [review]

**Pull request merged**